server/src/main/scala/io/delta/sharing/server/DeltaAction.scala

```diff
@@ -34,11 +34,53 @@ object DeltaAction {
   val readerVersion = 3
   val writerVersion = 2
 
+  case class PropertyAllowedValues(property: String, allowedValues: Seq[String])
+
+  /**
+   * These properties should be supported according to the readerFeatures.
+   * However, they are not supported by the Delta standalone reader at this point.
+   * Delta standalone reader should still try to read these tables given the properties are not set.
+   */
+
+  val columnMappingProperty: PropertyAllowedValues =
+    PropertyAllowedValues("delta.columnMapping.mode", Seq("none"))
+  val tablePropertiesWithDisabledValues: Seq[PropertyAllowedValues] =
+    Seq(columnMappingProperty)
+
+  val supportedTableFeatures: Set[TableFeature] = Set(
```

Comment:
is this and L56 used anywhere?

Reply:
not being used right now I will delete and reintroduce for the DV PR if needed